.gitignore

```diff
@@ -13,3 +13,4 @@ examples/**/static
 examples/**/dist
 cypress/videos
 cypress/screenshots
+.idea
```

Comment:
Please revert this

Reply:
I have read in some older PRs that this should be in user gitignore, but why is that so? Shouldn't the repository be responsible for what is ignored and what isn't?

Reply:
You can have a root `.gitignore` in your home directory for things specific to your tools.